atomicapp/nulecule/main.py

```diff
@@ -147,10 +147,19 @@ def genanswers(self, dryrun=False, answers_format=None, **kwargs):
         self.answers_format = answers_format or ANSWERS_FILE_SAMPLE_FORMAT
 
         # Check to make sure an answers.conf file doesn't exist already
-        answers_file = os.path.join(os.getcwd(), ANSWERS_FILE)
+        if location is None:
+            answers_file = os.path.join(os.getcwd(), ANSWERS_FILE)
+        else:
+            answers_file = os.path.join(location, ANSWERS_FILE)
```

Comment:
Double sigh.. problems like this make me want to retire. Ok we need to handle this for both the "in container" and "native" cases. That means if we are inside a container and the path is absolute then we need to translate the path like is done here: https://github.com/projectatomic/atomicapp/blob/master/atomicapp/nulecule/main.py#L72